rebar.config

```diff
@@ -34,7 +34,7 @@
 {shell, [{apps, [erldns]},
          {config, "erldns.config"}]}.
 
-{relx, [{release, {erldns, "1.0.0"},
+{relx, [{release, {erldns, "1.0.1"},
```

Comment:
I believe our versions our off here. Specifically I think the last release tagged for erldns is https://github.com/dnsimple/erldns/releases/tag/2.1.0 - this version should probably be 2.1.1, but you'll need to confirm first that all of the changes for the 2.1.0 tag are in main.

Reply:
I'll set it to track out tagged releases. Do we want to release it on [https://hex.pm/packages/erldns](https://hex.pm/packages/erldns)?

Changed in 1ed3590

Reply:
The following commits are not in 2.1.0, as they are in **main**.

```
😎  erldns git:(main) git --no-pager log --oneline 2.1.0..main
e2bdc4c (HEAD -> main, origin/main) * no lager output on exception propagation (#129)
ee82e0a * added additional handling of specific exceptions to propagate proper return codes for specific DNS queries (#128)
7c22d6d Externalized ingress UDP and TCP request timeouts to configuration variables (#127)
f229d68 Jsx 3.0.0 with maps (#126)
b50e071 jsx: v3.x uses maps unless proplists are explicitly forced (#110)
```

To get the diff run (at the time of writing this comment):

`git diff --color-words 2.1.0..main`

NOTE: If it's complaining likely you do not have the tag so fetch it via `git fetch --all --tags`

@aeden what do you think about starting a changelog?